content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md

```diff
@@ -7,14 +7,31 @@ menu:
     name: Release notes
 ---
 
+## v1.6.2 [2021-09-24]
+
+### Features
+
+- Add the `-template-id` property to the `GET /kapacitor/v1/tasks` request response. Adding this property helps to identify tasks that were created from a template.
+- For alert templates, row templates, and details templates, add support for third-party services that reject standard `json` (with carriage return at end) by adding the ability to use compact `json` in templates. (Also compacts Big Panda alert details to avoid Panda service error.) jsonCompact.
+
+### Bug fixes
+
+- Previously, errors could occur if a measurement string had special characters. Now, `expvar` string json encoding handles special characters, thanks @prashanthjbabu!
+- Update `jwt` dependencies of libraries to prevent this vulnerability: https://nvd.nist.gov/vuln/detail/CVE-2020-26160.
+- Switch to `github.com/golang-jwt/jwt` to prevent this vulnerability: https://nvd.nist.gov/vuln/detail/CVE-2020-26160.
+- Switch task service to use Flux formatter that preserves comments.
+- Previously, Kapacitor used a remote `influxdb` bucket (or 1.x database) to store Flux task run logs, and you had to manually create the database or bucket to store Flux logs. Now, Kapacitor automatically creates an InfluxDB 1.x database or InfluxDB 2.x bucket for Flux task logs. Note for InfluxDB 1.x, the `database` bucket name is created (rather than the 1.x format `database/rp` that includes the retention policy.
```

Comment:
This one is probably a feature rather than a fix (sorry if it was in the wrong place in the original kapa changelog!)

Reply:
Also is there a PR for this yet? https://github.com/influxdata/docs-v2/issues/3131

Reply:
thank you @lesam, looks like it moved to feature in change log, will update!

Reply:
updated PR to include https://github.com/influxdata/docs-v2/issues/3131